crates/bevy_mikktspace/src/generated.rs

```diff
@@ -263,7 +263,7 @@ pub unsafe fn genTangSpace<I: Geometry>(geometry: &mut I, fAngularThreshold: f32
         pTriInfos.as_mut_ptr(),
         piTriListIn.as_ptr(),
         geometry,
-        iNrTrianglesIn,
+        iNrTrianglesIn as i32,
```

Comment:
Isn't the data type inferred to be `i32` already? If not, what is the data type and why does the line number 255 exactly do when going below 0?